sources/packages/web/src/components/aest/students/modals/AssessApplicationChangeRequestModal.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <v-form ref="assessApplicationChangeRequestForm">
+    <modal-dialog-base :showDialog="showDialog" :title="title">
+      <template #content
+        ><error-summary :errors="assessApplicationChangeRequestForm.errors" />
+        <p class="my-4">
+          {{ subject }}
+        </p>
+        <v-textarea
+          label="Notes"
+          variant="outlined"
+          hide-details="auto"
+          v-model="note"
+          :rules="[checkNotesLengthRule]"
+          required
+        />
+        <p class="pt-1 brand-gray-text">
+          Notes will be visible to StudentAid staff and institutions. This will
+          not be shown to students.
+        </p>
+      </template>
+      <template #footer>
+        <check-permission-role :role="Role.StudentApproveDeclineAppeals">
+          <template #="{ notAllowed }">
+            <footer-buttons
+              :processing="loading"
+              :primaryLabel="primaryLabel"
+              @secondaryClick="cancel"
+              @primaryClick="assessChange"
+              :disablePrimaryButton="notAllowed"
+            />
+          </template>
+        </check-permission-role>
+      </template>
+    </modal-dialog-base>
+  </v-form>
+</template>
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { ref, defineComponent, computed } from "vue";
+import { useModalDialog, useRules } from "@/composables";
+import { ApplicationChangeRequestAPIInDTO } from "@/services/http/dto";
+import {
+  VForm,
+  Role,
+  ApplicationOfferingChangeRequestStatus,
+  ApplicationEditStatus,
+} from "@/types";
+import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
+
+export default defineComponent({
+  components: {
+    ModalDialogBase,
+    CheckPermissionRole,
+  },
+  setup() {
+    const note = ref("");
```

Comment:
I would recommend to default as undefined instead of empty string. `const note = ref<string>();`